src/components/DiaryCreate.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from '@emotion/styled'
+
+interface DiaryCreateProps {
+  newDiary: string
+  setNewDiary: (value: string) => void
```

Comment:
```suggestion
  onDiaryChange: (value: string) => void
```

diary 값 변경을 `DiaryCreate` 컴포넌트를 사용하는 상위 컴포넌트에 통지해주기 때문에 `set*`으로 시작하는 네이밍보다는 `on*`으로 시작하도록 변경해주는게 좋겠습니다.